ocbot/pipeline/handlers/airtable_request_handler.py

```diff
@@ -28,8 +28,11 @@ def api_calls(self):
         else:
             self.api_dict['user'] = self._event['Slack User']
 
-        matches = Airtable.find_mentors_with_matching_skillsets(self._event['Skillsets'])
-        ids = self.get_mentor_slack_ids(matches)
+        if 'Skillsets' in self._event:
```

Comment:
I wish we had assignment operator. :(

Reply:
Any chance while you're doing this you can add something for when someone requests a mentor?

Reply:
What do you mean?